libs/community/langchain_community/llms/azureml_endpoint.py

```diff
@@ -315,16 +323,32 @@ def format_response_payload(  # type: ignore[override]
         raise ValueError(f"`api_type` {api_type} is not supported by this formatter")
 
 
+class LlamaContentFormatter(CustomOpenAIContentFormatter):
+    """Deprecated: Kept for backwards compatibility
+
+    Content formatter for Llama."""
+
+    content_formatter: Any = None
+
+    def __init__(self) -> None:
+        super().__init__()
+        warnings.warn(
+            """`LlamaContentFormatter` will be deprecated in the future. 
+                Please use `CustomOpenAIContentFormatter` instead.  
```

Comment:
`LlamaContentFormatter` **has been** deprecated and will be removed in the future.  Please use `CustomOpenAIContentFormatter` instead.